contributor-guide/modules/ROOT/pages/contributors-faq.adoc

```diff
@@ -0,0 +1,129 @@
++
+Refer to xref:superproject/overview.adoc[] for a full description of the super-project.
+
+. *What exactly is a "modular arrangement"?*
++
+It's when the libraries can be used, and hence built, without creating the monolithic headers, without needing the root build files, and without needing the libraries to be arranged in the usual `root/libs/<name>` format.
+
+. *Will the move to Modular Boost change testing?*
++
+No, unless you want to. You will still be able to test with the current non-modular way. But you could also test the modular way.
+
+. *How will modular Boost work if there is no `root/libs/<name>` structure? Or is the structure still required?*
++
+The structure is still required for things like testing and documentation building.
+
+. *What happens to the numeric libraries that are currently sub-libraries, when sub-libraries are no longer supported?*
++
+The numeric libraries have been divided into four packages: libboost-numeric-conversion/, libboost-numeric-interval/, libboost-numeric-odeint/, libboost-numeric-ublas/.
+
+== New Libraries
+
+. *When considering developing a library for Boost, what is the right balance between ambitious functionality and limited functionality?*
++
+The keys to a successful development of a new library are to identify core functionality and start simple. Prioritize functionality that provides significant value to developers and aligns with Boost's goals of promoting high-quality, reusable libraries. Avoid adding unnecessary features that may increase complexity without adding much value. It's often beneficial to start with a simpler implementation that addresses a specific problem or use case effectively. A library with a narrow focus and a clear, intuitive API is more likely to be accepted by the Boost community than one that attempts to solve too many problems at once or introduces unnecessary complexity.
++
+In addition to this, consider future extensibility, performance, portability, current trends, and always remember you can seek community advice and feedback through the Boost mailing lists.
+
+== Testing
+
+. *What Boost libraries are the best examples of how to add Continuous Integration (CI) into the library testing process?*
++
+One of the best examples of how Continuous Integration (CI) is integrated into the testing process for a Boost library is the boost:asio[] library. This is a cross-platform pass:[C++] library for network and low-level I/O programming. Alternativel, boost:test[] is a library designed to support unit testing in pass:[C++]. It provides a framework for writing and running test cases, as well as utilities for organizing and reporting test results. boost:test[] leverages CI to ensure the correctness and reliability of its functionality across different platforms and compilers. 
```

Comment:
Any answer to this question will be very controversial. It'd be good to get some consensus on what good CI looks like. Or maybe ask Peter what he thinks. I think everyone is going to think they're scripts are the best because they implemented the things they thought were missing from other libraries. On the other hand, I believe we have a section about CI in the contributor guide, so we could refer to that.

Reply:
OK, I have watered down the question to provide "useful" rather than "best" examples, and added several more examples of libraries using CI.